src/feedspora/generic_client.py

```diff
@@ -199,12 +199,14 @@ def set_common_opts(self, account):
         '''
 
         # Tags
+        self._tags = []
         if 'tags' in account:
             self._tags = [
                 word.strip() for word in account['tags'].split(',')
             ]
 
         # Tag filtering options
+        self._tag_filter_opts = dict()
```

Comment:
Same here: the `_tag_filter_opts = dict()` should be in an `else`  statement.

Reply:
OK, but we need to work out the test functionality question below first, I think.